mkdocs.yml

```diff
@@ -61,7 +64,7 @@ nav:
     - December 2018: updates/2018-12.md
     - November 2018: updates/2018-11.md
     - October 2018: updates/2018-10.md
-  - Licensing: licensing.md
+  - Pricing: licensing.md
```

Comment:
Wouldn't this make the URL 'licensing'?

Reply:
Yes.. but there are still info about the licensing model on the page :)

Reply:
Yes but I didn't get the redirect to work as I wanted to I will take that in a separate PR